app/views/devise/registrations/new.html.erb

```diff
@@ -17,7 +17,7 @@
         required: true,
         input_html: { autocomplete: "new-password" } %>
       </div>
-        <%= f.button :submit, "Sign up", class: "btn btn-flat" %>
+        <%= f.button :submit, "Sign up", class: "" %>
```

Comment:
Is it possible to remove the class here if its empty

Reply:
```suggestion
        <%= f.button :submit, "Sign up", class: "btn btn-flat" %>
```